ci-operator/step-registry/idp/external-oidc/entraid/idp-external-oidc-entraid-chain.yaml

```diff
@@ -0,0 +1,9 @@
+chain:
+  as: idp-external-oidc-entraid
+  steps:
+  - ref: idp-external-oidc-entraid-prepare
+  - ref: idp-external-oidc
+  documentation: |-
```

Comment:
@xiuwang , currently (and maybe even for long time), no OCP external oidc env console login, which is an important checkpoint for OCP external oidc env, is verified at least once in automation. Given there are existing implemented steps for HCP external oidc env console login verification. I think we need to cover same in OCP too. Could you include them?
```
cucushift-hypershift-extended-external-oidc-console-app-redirect-uri-add
openshift-extended-web-tests-ext-oidc-cli-login
```

Reply:
The PR LGTM except this.

Reply:
If external-oidc-entraid-console-app-redirect-uri-[add|remove] is more proper?

Reply:
"external-oidc-entraid-console-app-redirect-uri-[add|remove]" is fine.